curriculum/challenges/english/07-scientific-computing-with-python/learn-tree-traversal-by-building-a-binary-search-tree/65cb40ee75b4f787b21066a8.md

```diff
@@ -1,87 +1,105 @@
 ---
-id: 655a5489c62889851c60ff4b
-title: Step 45
+id: 65cb40ee75b4f787b21066a8
```

Comment:
this step can probably be merged to the previous one

Reply:
49/50 combined now.